Microsoft/microsoft-intune/ingest/parser.yml

```
@@ -0,0 +1,57 @@
+name: microsoft-intune
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{json_event.message.time}}'
+        output_field: datetime
+  - name: set_common_fields
+  - name: action_properties
+    filter: "{{json_event.message.properties != null}}"
+
+stages:
+  set_common_fields:
+    actions:
+      - set:
+          '@timestamp': '{{parsed_date.datetime}}'
+          action.description: '{{json_event.message.properties.Description}}'
```

`event.reason` seems more suitable:
```suggestion
          event.reason: '{{json_event.message.properties.Description}}'
```